packages/vendure-plugin-campaign-tracker/src/entities/campaign.entity.ts

```
@@ -0,0 +1,34 @@
+import { DeepPartial, ID, VendureEntity } from '@vendure/core';
+import { Column, Entity, Unique } from 'typeorm';
+
+@Entity()
+@Unique(['channelId', 'code'])
+export class Campaign extends VendureEntity {
+  constructor(input?: DeepPartial<Campaign>) {
+    super(input);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)